krkn/scenario_plugins/kubevirt_vm_outage/kubevirt_vm_outage_scenario_plugin.py

```diff
@@ -0,0 +1,335 @@
+import logging
+import time
+from typing import Dict, Any, Optional
+
+import yaml
+from kubernetes.client.rest import ApiException
+from krkn_lib.k8s import KrknKubernetes
+from krkn_lib.models.telemetry import ScenarioTelemetry
+from krkn_lib.telemetry.ocp import KrknTelemetryOpenshift
+from krkn_lib.utils import log_exception
+
+from krkn.scenario_plugins.abstract_scenario_plugin import AbstractScenarioPlugin
+
+
+class KubevirtVmOutageScenarioPlugin(AbstractScenarioPlugin):
+    """
+    A scenario plugin that injects chaos by deleting a KubeVirt Virtual Machine Instance (VMI).
+    This plugin simulates a VM crash or outage scenario and supports automated or manual recovery.
+    """
+
```

Comment:
be sure to add a **get_scenario_types** function that returns the name of the scenario plugin 
https://krkn-chaos.dev/docs/contribution-guidelines/scenario_plugin_api/#get_scenario_types

like this format: https://github.com/krkn-chaos/krkn/blob/630dbd805b9540e2445422dcea10dfcdc390a827/krkn/scenario_plugins/application_outage/application_outage_scenario_plugin.py#L92-L93

Reply:
along with that, the abstract plugin couldn't find your file because there's not an init file in the new folder you added. Are you able to add a blank ___init__.py_ file to this folder